casepro/backend/tests/test_junebug.py

```diff
@@ -573,3 +737,101 @@ def request_callback_2(request):
             {'address': '+2222'},
             {'address': '+3333'},
         ]))
+
+    def single_identity_callback(self, request):
+        headers = {'Content-Type': 'application/json'}
+        data = {
+            'count': 1,
+            'next': None,
+            'previous': None,
+            'results': [
+                {
+                    "id": "50d62fcf-856a-489c-914a-56f6e9506ee3",
+                    "version": 1,
+                    "details": {
+                        "addresses": {
+                            "msisdn": {
+                                "+1234": {}
+                            }
+                        }
+                    },
+                    "communicate_through": None,
+                    "operator": None,
+                    "created_at": "2016-06-23T13:15:55.580070Z",
+                    "created_by": 1,
+                    "updated_at": "2016-06-23T13:15:55.580099Z",
+                    "updated_by": 1
+                },
+            ]
+        }
```

Comment:
We seem to have this data structure (or similar) in a few places. Should we maybe refactor it into a `make_identity()` helper?